.travis.yml

```diff
@@ -44,6 +44,31 @@ before_install:
 
 install:
   - yarn install --no-lockfile --non-interactive
+  include:
+    # runs linting and tests with current locked deps
+    - stage: "Tests"
+      name: "Tests"
+      script:
+        - npm run lint:hbs
+        - npm run lint:js
+        - npm test
+
+    - stage: "Additional Tests"
+      name: "Floating Dependencies"
+      install:
+        - npm install --no-package-lock
+      script:
+        - npm test
+
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.8
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.12
+    - env: EMBER_TRY_SCENARIO=ember-release
+    - env: EMBER_TRY_SCENARIO=ember-beta
+    - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
+    - env: EMBER_TRY_SCENARIO=ember-classic
```

Comment:
I think these changes to the `.travis.yml` are unnecessary, and they may have actually prevented Travis from running/failing this build. Let's revert the changes to this file and see if the build runs and goes green.

We're using `yarn` on this project (rather than the `npm` client). So, for example, we run the lint task in CI with the `yarn lint:js` command directly below. I'm not sure what tools auto-generated what here, so please let me know if need some of these changes.